pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/distributed.tsx

```diff
@@ -121,5 +121,18 @@ export default function (props: GraphDashboardProps) {
         }
       </Axis>
     </LineGraph>,
+
+    <LineGraph title="Mean Clock Offset" sources={nodeSources}>
+      <Axis label="nanos">
+        <Metric name="cr.node.clock.offset.meannanos" title="Mean Clock Offset" nonNegativeRate />
```

Comment:
looks like a typo in the metric name: it's `cr.node.clock-offset.meannanos`. https://github.com/cockroachdb/cockroach/blob/3dbe1dae8e734a27df53591f882efcd1543de9cd/pkg/rpc/clock_offset.go#L48